tests/SprykerEcoTest/Zed/Adyen/Business/Facade/AdyenFacadeExecutePostSaveHookTest.php

```diff
@@ -0,0 +1,256 @@
+
+use Generated\Shared\DataBuilder\AdyenApiResponseBuilder;
+use Generated\Shared\Transfer\AdyenApiMakePaymentResponseTransfer;
+use Generated\Shared\Transfer\AdyenApiResponseTransfer;
+use SprykerEco\Zed\Adyen\Dependency\Facade\AdyenToAdyenApiFacadeBridge;
+use SprykerEcoTest\Zed\Adyen\Business\BaseSetUpTest;
+
+/**
+ * @group Functional
+ * @group SprykerEco
+ * @group Zed
+ * @group Adyen
+ * @group Business
+ */
+class AdyenFacadeExecutePostSaveHookTest extends BaseSetUpTest
+{
+    /**
+     * @var string
+     */
+    protected const PROCESS_NAME_ADYEN_CREDIT_CARD = 'AdyenPayPal01';
+
+    /**
+     * @var string
+     */
+    protected const MAKE_PAYMENT_PAY_PAL_REQUEST_TYPE = 'MakePayment[PayPal]';
+
+    /**
+     * @var array
+     */
+    protected const MAKE_PAYMENT_RESPONSE_DETAILS = ['details'];
+
+    /**
+     * @var string
+     */
+    protected const MAKE_PAYMENT_RESPONSE_PAYMENT_DATA = 'payment data';
+
+    /**
+     * @var string
+     */
+    protected const OMS_STATUS_NEW = 'new';
+
+    /**
+     * @var string
+     */
+    protected const ADYEN_PAYMENT_STATUS_AUTHORISED = 'Authorised';
+
+    /**
+     * @var string
+     */
+    protected const ADYEN_PAYMENT_STATUS_REFUSED = 'Refused';
+
+    /**
+     * @var string
+     */
+    protected const ADYEN_PAYMENT_STATUS_ERROR = 'Error';
+
+    /**
+     * @var string
+     */
+    protected const ADYEN_PAYMENT_STATUS_CANCELLED = 'Cancelled';
+
+    /**
+     * @var string
+     */
+    protected const ERROR_TYPE_PAYMENT_FAILED = 'payment failed';
+
+    /**
+     * @var int
+     */
+    protected const ERROR_CODE_PAYMENT_FAILED = 399;
+
+    /**
+     * @return void
+     */
+    public function testExecutePostSaveHookShouldNotProcessErrorsWhenResponsIsSuccessAndDoesNotHaveRefusalStatus(): void
+    {
+        //Arrange
+        $adyenApiResponseTransfer = (new AdyenApiResponseBuilder([
+                AdyenApiResponseTransfer::IS_SUCCESS => true,
+            ]))
+            ->withMakePaymentResponse([
+                AdyenApiMakePaymentResponseTransfer::RESULT_CODE => static::ADYEN_PAYMENT_STATUS_AUTHORISED,
+                AdyenApiMakePaymentResponseTransfer::DETAILS => static::MAKE_PAYMENT_RESPONSE_DETAILS,
+                AdyenApiMakePaymentResponseTransfer::PAYMENT_DATA => static::MAKE_PAYMENT_RESPONSE_PAYMENT_DATA,
+            ])
+            ->build();
+
+        $adyenToAdyenApiFacadeBridgeMock = $this->createMock(AdyenToAdyenApiFacadeBridge::class);
+        $adyenToAdyenApiFacadeBridgeMock->method('performMakePaymentApiCall')
+            ->willReturn($adyenApiResponseTransfer);
+
+        $facade = $this->createFacade($adyenToAdyenApiFacadeBridgeMock);
+
+        $orderTransfer = $this->setUpCommandTest(
+            static::PROCESS_NAME_ADYEN_CREDIT_CARD,
+            static::OMS_STATUS_NEW,
+        );
+
+        $checkoutResponseTransfer = $this->tester->createCheckoutResponseTransfer();
+
+        //Act
```

Comment:
```suggestion
        // Act
```
same for other cases across this class